src/react-components/ErrorView.tsx

```diff
@@ -1,12 +1,44 @@
+import '../styles/errorView.css';
+import { NavLink, useHistory } from 'react-router-dom';
+
 export const ErrorView = ({ error }: {error: string}) => {
   let displayedError;
 
   if (error.length > 0) {
-    // TO DO: update error message text
     displayedError = "We seem to be having technical issues. Please try again later."
   } else {
-    displayedError = "404: Page not found. Please click the logo above to return home."
+    displayedError = "We can't seem to find the page you're looking for. You can head over to our homepage or recheck if you used the right address."
+  }
+
+  let displayedErrorImage;
+
+  if (error.length > 0) {
```

Comment:
We can make this code more DRY by combining these two conditionals - I can include this with my current branch changes in progress.